src/js/actions/event/data.js

```diff
@@ -0,0 +1,47 @@
+export const GET_EVENT = 'GET_EVENT';
+export const GET_EVENT_REQUEST = 'GET_EVENT_REQUEST';
+export const GET_EVENT_SUCCESS = 'GET_EVENT_SUCCESS';
+export const GET_EVENT_FAILURE = 'GET_EVENT_FAILURE';
+
+const event1 = {
+  event_id: 1,
+  host_user_id: 1,
+  name: 'Lounge party',
+  description: 'Celebrating life',
+  is_poll: true,
+  is_edited: false,
+  _invitees: [2],
+  _what: ['dancing', 'skydiving'],
+  _where: ['Forest', 'Camping'],
+  _when: ['2017-01-03T00:00:00.000Z', '2017-02-14T00:00:00.000Z']
```

Comment:
@heron2014 I hadn't thought about that.  I reckon the `_when: [{ date: ..., time: ...}]` format will be temporary.  Initially the date and time values will be separate, because we have separate inputs for them.  But the idea is to combine them into one timestamp eventually, before sending off to the server.  So the `_when: ['2017-01-03T00:00:00.000Z', '2017-02-14T00:00:00.000Z']` structure is the final version